src/hermes/commands/harvest.py

```diff
@@ -29,25 +37,14 @@ def harvest_cff(click_ctx: click.Context, ctx: HermesContext):
         return 1
     else:
         cff_dict = load_cff_from_file(cff_file)
-        if not validate(cff_dict):
+        if not validate(cff_file, cff_dict):
             return 1
         else:
```

Comment:
See above (reduce complexity, get rid for `else`)

Reply:
Thanks again, fixed in  2e709b601cae318ec72c93a9b78610a1db3b5cd1.